dspace-api/src/main/java/org/dspace/eperson/dao/impl/GroupDAOImpl.java

```diff
@@ -64,16 +64,33 @@ protected GroupDAOImpl()
         addMetadataSortQuery(queryBuilder, sortMetadataFields, null);
 
         Query query = createQuery(context, queryBuilder.toString());
+        if (pageSize > 0) {
+        	query.setMaxResults(pageSize);
+        }
+        if (offset > 0) {
+        	query.setFirstResult(offset);
+        }
         for (MetadataField sortField : sortMetadataFields) {
             query.setParameter(sortField.toString(), sortField.getID());
         }
         return list(query);
     }
 
+//    @Override
+//    public List<Group> findAll(Context context) throws SQLException {
+//    	return findAll()
+//    }
```

Comment:
I'm going to remove it.
I'm also going to rename the endpoints in groups as decided for now in the today call